packages/main/src/BusyIndicator.js

```diff
@@ -0,0 +1,101 @@
+import UI5Element from "@ui5/webcomponents-base/src/UI5Element.js";
+import Bootstrap from "@ui5/webcomponents-base/src/Bootstrap.js";
+import BusyIndicatorTemplateContext from "./BusyIndicatorTemplateContext.js";
+import BusyIndicatorRenderer from "./build/compiled/BusyIndicatorRenderer.lit.js";
+import Icon from "./Icon.js";
+
+// Styles
+import busyIndicatorCss from "./themes/BusyIndicator.css.js";
+
+// all themes should work via the convenience import (inlined now, switch to json when elements can be imported individyally)
+import "./ThemePropertiesProvider.js";
+import BusyIndicatorType from "./types/BusyIndicatorType.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-busy-indicator",
+	usesNodeText: true,
```

Comment:
This is no longer in the framework, you should merge from master and check some other web component (Button.js) -> create a default slot